src/metpy/calc/indices.py

```diff
@@ -323,7 +324,7 @@ def significant_tornado(sbcape, surface_based_lcl_height, storm_helicity_1km, sh
 @exporter.export
 @preprocess_xarray
 @check_units('[pressure]', '[speed]', '[speed]', '[length]', '[speed]', '[speed]')
-def critical_angle(pressure, u, v, heights, stormu, stormv):
+def critical_angle(pressure, u, v, height, stormu, stormv):
```

Comment:
Is this a signature to keep as-is? The two things that stood out to me as unexpected (based on the new quasi-standard) are:
- having the two different velocity component pairs separated by the height argument
- `stormu` and `stormv` might be better as `u_storm` and `v_storm`

Reply:
Done.